analytics.go

```diff
@@ -143,7 +143,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	c.startLoop()
```

Comment:
Actually based on all the discussions we had today and the directions we're going I don't think this comment is relevant anymore.